tools/pot/openvino/tools/pot/graph/graph_utils.py

```diff
@@ -29,7 +30,7 @@ def load_graph(model_config, target_device='ANY'):
 
     if target_device in special_transform_devices:
         network = ie.read_network(model=xml_path, weights=bin_path)
```

Comment:
should not it use IR FE to read model? in this case `ng.function_from_cnn` is not needed

Reply:
`apply_pot_transformations` cannot work with `ng.function_from_cnn(network)` because they are from different APIs